src/_includes/partials/components/filters.njk

```diff
@@ -0,0 +1,39 @@
+{% set filterableOptions = { projects: [], types: [] } %}
```

Comment:
As per the designs, filters must include (in order):

- Topic
- Type
- Project
- Language
- Publisher

Reply:
For the language, I just hardcoded the options as when I loop through resources with specific language, I would not be able to populate this object with what we want.